docs/assets/0123-import-scripts/README.md

```diff
@@ -0,0 +1,142 @@
+# Welcome to 0123-import-scripts!
+
+This is a template repository that can be used for the archiving process of a big dataset at the end of a research 
+project's lifetime. Download and unpack this repository from the 
+[excel2xml documentation page](https://docs.dasch.swiss/latest/DSP-TOOLS/dsp-tools-excel2xml/).  
+
+In this README, you will learn how to write a Python script for preparing data for an import into DSP.  
+
+Featuring: 
+
+- first steps with Visual Studio Code
+- the module `excel2xml` of dsp-tools
+- the benefits of Version Control with Git
+- the benefits of the Debugging Mode
+- extras: OpenRefine, Git GUIs, regexr
+
+
+# First steps with Visual Studio Code (VSC)
+[**Visual Studio Code**](https://code.visualstudio.com/download) is the industry standard IDE, free to use, and 
+recommended for your daily work at DaSCH. Recommended extensions to install from within VSC: 
+
+- redhat.vscode-xml
+- ms-python.python
+- ms-python.vscode-pylance 
+- zainchen.json
+- nickdemayo.vscode-json-editor
+- visualstudioexptteam.vscodeintellicode
+- redhat.vscode-xml
+- dotjoshjohnson.xml
+
+
+## Initialize Git
+Open this repository in Visual Studio Code, change to the "Source Control" tab, and click on "Inizialize Repository":
+
+![git init](assets/git-init.png)
+
+Stage all changes, write "init" as commit message, and commit all changes:
+
+![git commit](assets/git-commit.png)
+
+You now have the option "Publish Branch". This is to synchronize your local work with a GitHub repository on 
+[https://github.com/dasch-swiss/](https://github.com/dasch-swiss/). For this purpose, replace `0123` by your project's 
+shortcode, and `import` by your project's shortname. This is especially recommended for big projects where you spend 
+weeks/months on, when you might want to have a backup, or when you want to invite colleagues for collaboration or a code
+review.
+
+
+## Choose a Python interpreter
+Open `import-script.py`. You can now choose a Python interpreter by clicking on the Version number on the bottom right.
+You can either work with the global (system-wide) Python, or you can create a 
+[virtual environment](https://python.land/virtual-environments) for your project. If you don't know which one to choose, 
+take the one installed via Homebrew, which is located in `/usr/local/Cellar`. Probably you already have a symlink 
+(`/usr/local/bin/python3` or `/usr/bin/python3`) that redirects to `/usr/local/Cellar`. The only thing that you 
```

Comment:
again, not sure what the target audience is - but like this it's 100% Mac-centric, whereas DSP-Tools is actually intended to be OS independent